tests/model_fields/test_imagefield.py

```diff
@@ -194,6 +194,12 @@ def test_pickle(self):
         loaded_p = pickle.loads(dump)
         self.assertEqual(p.mugshot, loaded_p.mugshot)
 
+    def test_defer(self):
+        self.PersonModel.objects.create(name='Joe', mugshot=self.file1)
+        qs = list(self.PersonModel.objects.defer('mugshot'))
```

Comment:
The extra queries could be generated in this line (199), as this is where the instancies are created, so so you should check the number of queries is one.